website/docs/advanced/builder.md

```diff
@@ -0,0 +1,190 @@
+ - if configuring with the `--proposer-settings-file` or `--proposer-settings-url` flag with no builder settings but providing the `--enable-builder` flag instead. Optionally, you can also add the `--suggested-gas-limit` to adjust the default gas limit for the builder, this only applies with `--enable-builder`.
+ - if configuring with the `--suggested-fee-recipient` and `--enable-builder` flags. **note:**  `--proposer-settings-file` or `--proposer-settings-url` flags with builder settings will override values provided from `--suggested-fee-recipient` and `--enable-builder`flags.
+
+:::info
+
+Validators updated through the [Keymanager-API's](../how-prysm-works/keymanager-api.md) fee recipient APIs will take on the default `proposer-settings` provided.
+
+If the `--enable-builder` flag is used without providing `--suggested-fee-recipient`, `--proposer-settings-file`, or `--proposer-settings-url` it will override builder settings from the db if proposer settings are saved, or it will set default builder settings and only save to the db if fee-recipient settings are saved through the keymanager APIs.
+
+The validator client utilizes `proposer-settings` to interact with the beacon node's [Beacon API](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/registerValidator). Subsequently, the beacon node calls the builder by making use of the [Builder API](https://ethereum.github.io/builder-specs/#/Builder/registerValidator).
+
+Registration against the builder will only occur for active validators. Registration for eligible validators occurs at the beginning of the validator clients execution and at the middle of each epoch. It is important to note that the success of the API is not guaranteed, and the client will attempt registration again at the middle of each epoch.
+
+The beacon node must also be configured to enable the builder via the `--http-mev-relay` flag.
+
+:::
+
+## 2. Beacon Node: connect to the builder
+
+To use a builder the beacon node needs to start with the following configuration:
+
+- `--http-mev-relay` flag pointed to a MEV-boost instance or running relay (5. Builder: connected via relay URL).
+
+Each relay's URL will correspond to a specific network and will need to be chosen accordingly, i.e. running a beacon node on mainnet will require the mainnet relay.
+
+### Circuit breaker
+
+The circuit breaker is a safety feature for falling back to local execution when using a builder.
+This occurs when the builder or client using the builder endpoints encounters issues that cause missed blocks. 
+By default, the circuit breaker will be triggered after 3 slots are consecutively missed or 5 slots are missed in an epoch, but this can be configured through the `--max-builder-consecutive-missed-slots` and `max-builder-epoch-missed-slots` flags.
+
+## Registration cache
```

Comment:
Maybe move to the FAQ ? This feels a bit too specific to be in the hot path of instructions